src/l10n/ru.js

```diff
@@ -0,0 +1,354 @@
+            'Always show search section'          : 'Всегда показывать поле поиска',
+            'Notifications'                       : 'Уведомления',
+            'Send Emails for'                     : 'Посылать e-meil в случае',
+            'Security issues'                     : 'Инцидентов по безопасности',
+            'Passwords shared with me'            : 'Предоставления мне паролей',
+            'Show Notifications for'              : 'Показывать уведомления в случае',
+            'Other errors'                        : 'Других ошибок',
+            'Danger Zone'                         : 'Опасная зона',
+            'Reset all settings'                  : 'Сбросить все настройки',
+            'Reset'                               : 'Сброс',
+            'Delete everything'                   : 'Удалить всё',
+            'DELETE EVERYTHING'                   : 'ВСЁ БУДЕТ УДАЛЕНО',
+            'Account reset requested'             : 'Сформирован запрос на сброс аккаунта',
+            'View'                                : 'Просмотр',
+            'Default'                             : 'Обычный',
+            'Advanced'                            : 'Расширенный',
+            'There is nothing here'               : 'Здесь ничего нет',
+            'Click on "+" to add something'       : 'Для добавления нажмите на "+"',
+            'Deleted items will appear here'      : 'Здесь отображаются удалённые объекты',
+            'No passwords were shared with you'   : 'Ещё никто не поделился с вами',
+            'You did not share any passwords'     : 'Вы ещё никому не предоставляли доступ к вашим паролям',
+            'Your favorites will appear here'     : 'В данном разделе отображаются избранные объекты',
+            'Better check the other sections'     : 'Проверьте другие разделы',
+            'That\'s probably a good sign'        : 'Возможно, это хороший знак',
+            'Go to {href}'                        : 'Переход на {href}',
+            'Figure {count}: {title}'             : 'Цифр {count}: {title}',
+            'Unable to fetch page: {message}.'    : 'Невозможно открыть страницу: {message}.',
+            'Not Found'                           : 'Не найдено',
+            'Missing something or found an error?': 'Не можете что-то найти или столкнулись с ошибкой?',
+            'Tell us!'                            : 'Свяжитесь с нами!',
+            'Network error'                       : 'Ошибка сети',
+            'Unable to load {module}'             : 'Не удаётся загрузить модуль {module}',
+            'Sort by name'                        : 'Сортировать по имени',
+            'Sort by modified date'               : 'Сортировать по времени изменения',
+            '{passwords} passwords'               : 'паролей: {passwords}',
+            '1 password'                          : '1 пароль',
+            '{folders} folders'                   : 'папок: {folders}',
```

Comment:
I inverted the order due to improper endings of words in Russian in this very case. It can not be fixed because of multiple variants of endings for plurals. But anyone understand what is written in case of the standard order:

295  '{passwords} passwords'  :  '{passwords} паролей',
297  '{folders} folders'               : '{folders} папок',

May be it is even better.